src/main/java/frc/robot/commands/drive/DriveFieldOriented.java

```diff
@@ -16,13 +16,21 @@
 import frc.robot.Constants;
 import frc.robot.RobotContainer;
 import frc.robot.subsystems.Drive;
+import java.util.function.Supplier;
 
 /** Drives the robot in field-oriented mode. */
 public class DriveFieldOriented extends Command {
   private final Drive drive;
+  private final Supplier<Double> xScaledSupplier, yScaledSupplier, zScaledSupplier;
```

Comment:
Rename these to xSupplier, ySupplier, etc. In this context we don't care that the controller axes are scaled.